mailpoet/assets/js/src/automation/integrations/mailpoet/steps/add_tags/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { __ } from '@wordpress/i18n';
+import { tag } from '@wordpress/icons';
+import { StepType } from '../../../../editor/store';
+
+export const step: StepType = {
+  key: 'mailpoet:add-tag',
+  group: 'actions',
+  title: __('Add tag', 'mailpoet'),
+  description: __('Add a tag or multiple tags to a subscriber.', 'mailpoet'),
+  subtitle: () =>
+    __(
```

Comment:
if you have an automation with such a trigger, but premium is disabled, you would see this message.

Reply:
👍 Should someone proofread it? We could even go further and use https://github.com/mailpoet/mailpoet/blob/3279fb1154a98f19b5b09d925cd24d8159bc7723/mailpoet/assets/js/src/common/premium_modal/upgrade_info.ts in some way.